packages/contracts/contracts/L1/L2OutputOracle.sol

```diff
@@ -189,10 +191,14 @@ contract L2OutputOracle is Initializable, Semver {
         uint256 _l1BlockNumber,
         uint256 _bondAmount
     ) external payable {
-        require(
-            msg.sender == VALIDATOR_POOL.nextValidator(),
-            "L2OutputOracle: only the next selected validator can submit output"
-        );
+        address nextValidator = VALIDATOR_POOL.nextValidator();
```

Comment:
Same as replaceL2Output function, it looks good to restrict the sender to COLOSSEUM.

```suggestion
require(
            msg.sender == COLOSSEUM,
            "L2OutputOracle: only the colosseum contract can submit an output"
        );
```